### DISCUSSION TITLE
Steam Deck support possible?

### DISCUSSION THREAD

**Discussion by rafrafek** [Rem0o/FanControl.Releases#2303] [Q&A]
Hello,

I'm using Steam Deck with Windows 11 and I'm trying to use Fan Control to control the Steam Deck's fan. Currently the program detects one sensor but zero fans, but I think it just needs some code adjustments. 

I think Fan Control uses the same library as Steam Deck Tools, which is LibreHardwareMonitor.

Does it mean Steam Deck could be supported in Fan Control?

Steam Deck Tools implementation: 

https://github.com/ayufan/steam-deck-tools/blob/main/FanControl/FanController.cs

and probably most important:

https://github.com/ayufan/steam-deck-tools/blob/main/CommonHelpers/Vlv0100.cs

**Comment by Rem0o** [comment]
The FanController portion could very well be ported onto a [plugin](https://github.com/Rem0o/FanControl.Releases/wiki/Plugins).
